lib/davinci_dtr_test_kit/client_groups/light_ehr/dtr_light_ehr_user_response_test.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../../urls'
+require_relative '../../tags'
+
+module DaVinciDTRTestKit
+  # noinspection RubyJumpError,RubyResolve
+  class DTRLightEHRUserResponseTest < Inferno::Test
+    include URLs
+    id :dtr_light_ehr_user_response
+    title 'Checks for a valid user response at the supported payer endpoint'
+    description %(
+      This test verifies that a user-provided response is valid
+      when a request is made to the supported payer endpoint.
+    )
+
+    input :unique_url_id, type: :text
+    input :user_response, type: :textarea, optional: true
+
+    run do
+      if user_response.nil?
+        pass 'No user response provided, skipping validation and using default response.'
+        return
+      end
+
+      load_tagged_requests(SUPPORTED_PAYER_TAG)
+      request_record = requests.find { |r| r.tags.include?(SUPPORTED_PAYER_TAG) }
```

Comment:
shouldn't need this line - the tagged requests should be put into the `request` variable, so you can just use that.